### DISCUSSION TITLE
Propagation of trace-id from caduecus to webhook endpoints.

### DISCUSSION THREAD

**Discussion by Sachin4403** [xmidt-org/xmidt#73] [Ideas]
Hi team,

After receiving events from Talaria, Caduceus will push the events to the webhooks queue and webhooks will send the same to webhook endpoints with configured retry. but we are not sending the tracing information from Caduceus to webhooks, due to this a new trace-id is generated at the receiver side and it is very hard to track the same event. Can we propagate the same tracing information to webhooks?

cc: @schmidtw

**Comment by denopink** [comment]
related to https://github.com/xmidt-org/talaria/issues/284

**Comment by denopink** [comment]
cc: @renaz6
